.github/scripts/common.py

```diff
@@ -4,16 +4,23 @@
 from fabric.api import *
 import requests
 from local_flags import RUN_LOCAL, local_fsim_dir
+from ci_variables import ci_firesim_dir
 
 # Reuse manager utilities
 sys.path.append(local_fsim_dir + "/deploy/awstools")
 from awstools import get_instances_with_filter
 
+# Github URL related constants
+gh_repo_path        = "firesim/firesim"
```

Comment:
This can be `$GITHUB_REPOSITORY`. See https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
I didn't want to introduce yet another ci_variable dep, because i'd have to sensibly handle the `run_local` case beyond hardcoding the string (which brings us back here). 

I could query the remotes and look for one named origin.

Reply:
We don't support running scripts locally that query the GH API (we set the `GITHUB_TOKEN/PERSONAL_ACCESS_TOKEN` to `0`), so it should be fine to add these in.

Reply:
> We don't support running scripts locally that query the GH API

Why not?

Reply:
I think the main barrier is providing an easy way to share the secrets (firesim.pem, and the personal access tokens) are the thorny ones really. Many of the other "secrets" , like aws region, needed be secret. 

I'd imagine it was the same for Abe, but when i initially developed some of this stuff i just set all the env variables manually.